clustering/web/cache/src/main/java/org/wildfly/clustering/web/cache/sso/CompositeSSOManager.java

```diff
@@ -22,10 +22,11 @@
 package org.wildfly.clustering.web.cache.sso;
```

Comment:
A number of header files, as in this one, have old dates in them...

Reply:
The dates reflect the year they were created.

Reply:
So they are not updated when changes are made, OK.